client/rpc/dex_client.go

```diff
@@ -11,6 +11,9 @@ const (
 	AccountStoreName = "acc"
 	TokenStoreName   = "tokens"
 	ParamABCIPrefix  = "param"
+	TimeLockMsgRoute = "timelock"
+	QueryTimeLocks   = "timelocks"
```

Comment:
To keep the code style consistent,  `QueryTimeLocks ` and `QueryTimeLock ` please just use it, don't define the constant.